src/shared/components/plots/MultipleCategoryBarPlotUtils.ts

```diff
@@ -79,7 +80,13 @@ export function makePlotData(
     );
     return data;
 }
+interface TotalSumItem {
+    majorCategory: string;
+    sum: number;
+    minorCategory: { name: string; count: number; percentage: number }[];
+}
 
+export { TotalSumItem };
```

Comment:
we can delete this and just import and use the one you made here
https://github.com/cBioPortal/cbioportal-frontend/blob/1c2a8f4bfa8983a34816e99d4bbb9beb59658d66/src/pages/groupComparison/MultipleCategoryBarPlot.tsx#L63